src/panels/config/users/dialog-user-detail.ts

```diff
@@ -0,0 +1,209 @@
+import "@material/mwc-button";
+import "@polymer/paper-input/paper-input";
+import "@polymer/paper-tooltip/paper-tooltip";
+import {
+  CSSResult,
+  customElement,
+  html,
+  LitElement,
+  property,
+  TemplateResult,
+  css,
+} from "lit-element";
+import "../../../components/entity/ha-entities-picker";
+import "../../../components/user/ha-user-picker";
+import { PolymerChangedEvent } from "../../../polymer-types";
+import { haStyleDialog } from "../../../resources/styles";
+import { HomeAssistant } from "../../../types";
+import { UserDetailDialogParams } from "./show-dialog-user-detail";
+import { createCloseHeading } from "../../../components/ha-dialog";
+import { GROUPS, SYSTEM_GROUP_ID_USER } from "../../../data/user";
+
+@customElement("dialog-user-detail")
+class DialogUserDetail extends LitElement {
+  @property() public hass!: HomeAssistant;
+  @property() private _name!: string;
+  @property() private _group?: string;
+  @property() private _error?: string;
+  @property() private _params?: UserDetailDialogParams;
+  @property() private _submitting: boolean = false;
+
+  public async showDialog(params: UserDetailDialogParams): Promise<void> {
+    this._params = params;
+    this._error = undefined;
+    this._name = params.entry.name || "";
+    this._group = params.entry.group_ids[0];
```

Comment:
Why do this instead of just looking it up from `this._params` in render method ? That way you keep a single source of truth.

Reply:
Because the user can change the name, and this way we have the original and the user-edited value